file_catalog/config.py

```diff
@@ -0,0 +1,51 @@
+
+from ConfigParser import SafeConfigParser
+
+import os
+import ast
+
+class Config(dict):
+    def __init__(self, path):
+        self.path = path
+        self.list_cache = {}
+
+        # read file
+        tmp = SafeConfigParser()
+        tmp.read(path)
+        self.__config_options_dict(tmp)
+
+    def __config_options_dict(self, config):
```

Comment:
Prefer a single leading underscore for non-public methods:
https://www.python.org/dev/peps/pep-0008/#id45